### PR TITLE
fix: spacing in signup

### PR DESCRIPTION
**Pull request by hbjORbj**
## What does this PR do?

- Fix top margin

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] N/A - I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: apps/web/modules/signup-view.tsx [hunk 1]
```diff
@@ -619,57 +619,51 @@ export default function Signup({
               </div>
             </div>
           </div>
-          <div className="border-subtle lg:bg-subtle mx-auto mt-24 w-full max-w-2xl flex-col justify-between rounded-l-2xl pl-4 dark:bg-none lg:mt-0 lg:flex lg:max-w-full lg:border lg:py-12 lg:pl-12">
-            {IS_CALCOM && (
```
Comment (hbjORbj) [inline apps/web/modules/signup-view.tsx:623]: remove

Comment (sean-brydon) [reply apps/web/modules/signup-view.tsx:623]: Are we still wanting to show this even if a user is self hosted?

Comment (sean-brydon) [reply apps/web/modules/signup-view.tsx:623]: I think we need to still have this IS_CALCOM check for social proof 

CC: @PeerRich

Comment (keithwillcode) [reply apps/web/modules/signup-view.tsx:623]: Yeah we need to keep this for self-hosters

**Review by keithwillcode — CHANGES_REQUESTED** [review]

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**